pkg/cli/client.go

```diff
@@ -70,6 +70,10 @@ func NewClientCommand() *cobra.Command {
 
 			ctx, span = telemetry.Start(ctx, "cmd.main")
 
+			if server == "" {
+				return fmt.Errorf("required flag(s) \"server\" not set")
```

Comment:
For some reason `_ = cmd.MarkFlagRequired("server")` doesn't work in the root CLI file. So I just wrote this by hand to ensure the argument is given.

Without this it blows up in a very hard to interpret way.